README.md

```diff
@@ -37,25 +37,33 @@ stringify(input, options).then((output) => {
 By default, this module uses Node's builtin `Promise` object, but if you prefer to use a different promise library, you can do so simply by overwriting the `Promise` property on the module.
 
 ```js
-'use strict'
-
-let stringify = require('csv-stringify-as-promised')
-let bluebird = require('bluebird')
+const stringify = require('csv-stringify-as-promised');
+const bluebird = require('bluebird');
 
-stringify.Promise = bluebird
+stringify.Promise = bluebird;
 
-let input = [ [ '1', '2', '3', '4' ], [ 'a', 'b', 'c', 'd' ] ]
+const input = [ [ '1', '2', '3', '4' ], [ 'a', 'b', 'c', 'd' ] ];
 
 stringify(input).then((output) => {
   // uses bluebird instead of Promise
 }).catch((err) => {
   // handle error
-})
+});
+```
+
+### Use your own csv-stringify library
```

Comment:
This is an interesting idea, but basically requires that the custom csv library follows the exact same shape with the exact same options as csv-stringify, so I don't see that this is ultimately a useful feature.

Reply:
ex. I can fork `csv-stringify` which will have the same interface

Reply:
I think the logic around this is pretty simple. If someone is forking csv-stringify already, they can just include a Promise implimentation and there's no need to use this module. 

That said, I no longer use this module and would be happy to transfer ownership to a new maintainer. So if you want to maintain it and make these changes, go for it. Just let me know what your npm username is.

Reply:
my NPM username is the same as the GitHub username - `oprogramador`

Reply:
Done.